x/lockup/keeper/lock.go

```diff
@@ -377,9 +379,28 @@ func (k Keeper) unlockMaturedLockInternalLogic(ctx sdk.Context, lock types.Perio
 		return err
 	}
 
+	// If the lock contains CL liquidity tokens, we do not send them back to the owner.
```

Comment:
Please have a direct test for this

Reply:
https://github.com/osmosis-labs/osmosis/pull/4884/commits/ef5db7a29a17940de4bc7935e1fb9625ac41149d